src/components/ShowList.css

```diff
@@ -55,3 +72,81 @@
   color: green;
   font-weight: 600;
 }
+.shopingList {
+  font-family: 'Capriola', sans-serif;
+  font-size: 35px;
+  margin: 5px;
+}
+.product-name {
+  margin: auto;
+  display: flex;
+}
+
+.product {
+  border: 4px solid;
+  width: 15rem;
+  margin-bottom: 8px;
+  text-align: left;
+  padding-left: 3px;
+  border-radius: 15px;
+  margin-right: 10px;
+  font-family: 'jaldi', sans-serif;
+}
+
+.product-name span {
+  align-self: center;
+}
+
+.filter-delete {
+  display: flex;
+  justify-content: center;
+}
+.clear-button {
+  align-self: center;
+  font-size: 25px;
+  counter-reset: black;
+}
+
+.logo {
+  background-color: #e93b81;
+  position: absolute;
+  transform: translate(-50%, -50%);
+  left: 50%;
+  top: 12%;
+}
+.paper-container {
+  background-color: #fafafa;
+  border-radius: 20px;
+  position: absolute;
+  top: 15%;
+  left: 21%;
+  width: 55%;
+}
```

Comment:
When I run the app on my computer, the card background of ShowList is cutting behind the navigation buttons a little, I think because they're both absolutely positioned and my screen size is too short. It might not be something we have time to fix as I know we're nearly at the end of the week, but if we don't perhaps we can tweak it later on just so it looks great on all screens 😊

Screenshot below (the item scroll is working perfectly FYI, so it doesn't stop me using the app)
![image](https://user-images.githubusercontent.com/74988392/132093705-1e7bbba9-92cf-4525-9ffa-848b4e5c1deb.png)